docker/gcp_pudl_etl.sh

```diff
@@ -44,7 +43,7 @@ function run_pudl_etl() {
         -n auto \
         --gcs-cache-path gs://internal-zenodo-cache.catalyst.coop \
         --etl-settings $PUDL_SETTINGS_YML \
-        --live-dbs test/validate
+        --live-dbs test/validate \
```

Comment:
non-blocking: Was this missing `\` breaking things?

Reply:
Yes. This is the fix that @bendnorman added on his tiny core-dump debugging PR that unfortunately didn't fix the core-dump, but did mean the script could actually run.